elastic/kibana
# [Security Assistant] Move security AI assistant button into global nav bar

## Summary

More changes are needed within the observability and search solution to close the issue fully.

Summarise your PR. If it involves visual changes include a screenshot or gif.

Move the security AI assistant button from the solution header bar into the global nav bar. This is part of the AI assistant unification initiative. 

### How to Test
- Start kibana
- Go to one of the security solution pages (e.g. attack discovery)
- AI assistant button should be in the global nav bar. Clicking it opens the assistant.

- The button can also be tested for security serverless deployment. It should look like the screenshot bellow.

### Checklist

Check the PR satisfies following conditions. 

Reviewers should verify this PR satisfies this list as well.


### Identify risks

Does this PR introduce any risks? For example, consider risks like hard to test bugs, performance regression, potential of data loss.

Describe the risk, its severity, and mitigation for each identified risk. Invite stakeholders and evaluate how to proceed before merging.


Classic:
![image](https://github.com/user-attachments/assets/b2a9c982-bc54-42f4-ab59-6f0c99d4d899)
![image](https://github.com/user-attachments/assets/1ae36af0-5d1a-4519-844a-563074646ddf)

Serverless:
![image](https://github.com/user-attachments/assets/345280df-0e70-4203-b0d8-48ad11753f74)
![image](https://github.com/user-attachments/assets/7425c886-4528-4987-a00a-48bdc71728c7)

Old:
<img width="1728" alt="image" src="https://github.com/user-attachments/assets/5ef568c6-2d31-47da-8f5f-87dfdf10cb5c">
